### ISSUE TITLE
Redirection for the 2020 Winter issue not available

### ISSUE THREAD

**Issue by BlueReZZ** [microsimulation/ijm#108]
**Describe the bug**
Since the infrastrcture redirections made via https://github.com/microsimulation/ijm/blob/master/nginx_configuration/rewrite.conf were written in the Summer of 2020 the Winter issue wasn't covered and needs adding

**Steps To Reproduce**
https://microsimulation.pub/V13_1/ijm.00210.pdf should redirect to https://microsimulation.pub/articles/00210, and all other articles in volume 13 issue 1 should redirect in a similar way to their respective article pages.

**Comment by BlueReZZ** [comment]
I suspect that we need a new line for V13_1 similar to:
```
location ~* "^/IJM/V12_3/.*(\d{5})\.pdf$" {
            return 301 http://microsimulation.pub/articles/$1;
        }
```

Further issues will only be published on the new site so this would be the last redirection needed

**Comment by BlueReZZ** [comment]
@erkannt can we have a look at this later?

**Comment by erkannt** [comment]
Redirect from microsimulation.org is incorrectly setup for the rewrites on the microsimulation.pub side of things.

The rewrites here expect the path to begin with `/IJM/` but .org strips it.

**Comment by erkannt** [comment]
rewrite rule for volume 1-1 was missing zero padding. fixed with 04f53cd

**Comment by erkannt** [comment]
`./tests/redirects.sh` passes, so I'm closing this issue but raising a new one for a bunch of other DOIs that are lacking redirects.
